src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -533,6 +533,7 @@ private string AskForCloudApplicationNameFromDeployedApplications(List<CloudAppl
             var title = "Select an existing AWS deployment target to deploy your application to.";
 
             var userInputConfiguration = new UserInputConfiguration<CloudApplication>(
+                app => app.DisplayName,
                 app => app.DisplayName,
                 app => app.DisplayName.Equals(deployedApplications.First().DisplayName))
             {

```

Comment:
```suggestion
            var userInputConfiguration = new UserInputConfiguration<CloudApplication>(
                idSelector: app => app.DisplayName,
                displaySelector: app => app.DisplayName,
                defaultSelector: app => app.DisplayName.Equals(deployedApplications.First().DisplayName))
            {
```

use labels, param values now have started to lose the context. Check other `UserInputConfiguration` constructor invocations too.

Reply:
Done